isis/src/base/apps/makeflat/main.cpp

```diff
@@ -391,10 +390,6 @@ void IsisMain() {
 
   cubeInitialized = false;
   for(currImage = 0; currImage < inList.size(); currImage++) {
-    if(Excluded(currImage)) {
-      excludedCnt ++;
-      continue;
```

Comment:
We probably want to leave the
```cpp
if(Excluded(currImage){
  continue;
}

Reply:
Why's that? Wouldn't the loop continue regardless?

Reply:
`continue` inside a for-loop will skip the remaining logic in the for-loop and move to the next iteration.  If Excluded(currImage) is true, we want to short-circuit the rest of the logic in the loop and move to the next image.

Reply:
Oh gotcha. That makes sense.